scripts/ci/verify-adoption-doc.py

```
@@ -0,0 +1,204 @@
+                print(f"    ✗ FAIL (exit {rc})")
+
+    total = overall_pass + overall_fail + overall_skip
+    print(f"\n{'═' * 60}")
+    print(f"Total: {total}   PASS: {overall_pass}   FAIL: {overall_fail}")
+
+    if failures:
+        print("\n┄ Drift detected — the following verify blocks failed: ┄\n")
+        for doc, h2, verify_hdr, code, out in failures:
+            print(f"┌── {doc} ── §{h2} ── {verify_hdr}")
+            print(textwrap.indent(code.rstrip(), "│  "))
+            print(f"│")
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)